src/test/rustdoc/raw-ident-eliminate-r-hashtag.rs

```diff
@@ -0,0 +1,19 @@
+// ignore-tidy-linelength
+
+pub mod internal {
+    pub struct r#mod;
+
+    /// See [name], [other name]
+    ///
+    /// [name]: mod
+    /// [other name]: ../internal/struct.mod.html
```

Comment:
What is this meant to test?
```suggestion
    /// [other name]: crate::internal::mod
```

Reply:
I wrote it too quickly and simply forgot to move it to intra-doc links. I wanted to test both the short and long versions.